### ISSUE TITLE
GetDirection() behaves differently when called from ImageFileReader than from Image

### ISSUE THREAD

**Issue by Roulbac** [SimpleITK/SimpleITK#691]
When I use an ImageFileReader, call the ReadImageInformation() method then call GetDirection(), the tuple that is output is column-major whereas when I call GetDirection() from the Image itself (after reader.Execute()), the tuple is row-major.
So I have to transpose my 3x3 Direction matrix when I call direction from reader rather than the Image itself.
I don't need to load the images, I just need to get their information.
Is this normal behavior?

**Comment by blowekamp** [comment]
Thank you for reporting this issue. I believe I have reproduced it:

```
import SimpleITK as sitk
img = sitk.Image(10,10,10,sitk.sitkFloat32)
img.SetDirection([0,1,0,0,0,1,1,0,0])
sitk.WriteImage(img, "test.nrrd")
reader = sitk.ImageFileReader()
reader.SetFileName("test.nrrd")
reader.ReadImageInformation()
print(img.GetDirection())
print(reader.GetDirection())
```

**Comment by Roulbac** [comment]
I use a workaround in my code for this, any idea how I can have a version check to see if the bug is fixed in the version my code runs? Is this fix in sitk 1.2.0 ?

**Comment by blowekamp** [comment]
The patch has been applied to the 1.2.0 release branch. The bug is contained in the 1.2.0 release. It will be fix in 1.2.1 and 1.3.0.

However in the "development" versions it will be a little more tricky. The bug was fixed in version "1.2.0.post17" in the release branch and "1.3.0.dev111" in the master branch. This version scheme is follows `distutils.version.StrictVersion`. 

Please share the logic you us!
